code/API_definitions/device-reachability-status-subscriptions.yaml

```diff
@@ -400,7 +397,7 @@ components:
         subscriptionExpireTime:
           type: string
           format: date-time
-          example: "2023-01-17T13:18:23.682Z"
+          example: 2023-01-17T13:18:23.682Z
```

Comment:
The "quotation" marks were added because of [this](https://github.com/camaraproject/DeviceStatus/issues/238) issue. So they all need to stay for date-time examples.